hbase-migration-tools/bigtable-hbase-replication/README.md

```diff
@@ -232,6 +214,57 @@ refer
 to [IncompatibleMutationAdapter](bigtable-hbase-replication-core/src/main/java/com/google/cloud/bigtable/hbase/replication/adapters/IncompatibleMutationAdapter.java)
 javadocs for more details.
 
+### Dry run mode
+
+It may be hard to determine if an application issues incompatible mutations.
```

Comment:
nit - "It may be hard to determine if an application issues incompatible mutations, especially if the HBase cluster and application are owned by different teams."